gradlew

```diff
@@ -0,0 +1,234 @@
+#!/bin/sh
```

Comment:
Could you remove gradle/wrapper, gradlew, gradlew.bat, and .idea files from the PR?

Reply:
Yep I tried deleting them, but the gradle/wrapper/gradle-wrapper.jar did not have the delete option :(. And yes I assumed that the tests failed because loginIB was not implemented.